.github/workflows/build.yml

```diff
@@ -115,11 +115,11 @@ jobs:
         path: ${{ steps.compress_server_binary.outputs.path }}
 
     - name: Build Wrapper
-      if: matrix.ghc == '8.10.5'
+      if: matrix.ghc == '8.10.5' && matrix.os != 'macOS-latest' || matrix.ghc == '8.10.4' && matrix.os == 'macOS-latest'
```

Comment:
Could we simply use GHC 8.10.4 to build the wrapper for all platforms until the next stable GHC version released?

Reply:
Oops, thank you for the review. Maybe that's better as GHC 8.10.5 also does not expose `allocateWrite` symbol (which causes some GHC-plugin failures).